recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.4.1" %}
+{% set version = "0.4.2" %}
 
 package:
   name: pandana
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pandana/pandana-{{ version }}.tar.gz
-  sha256: "bcdedb4eb21da0d1012e76c8af6ad2c31124bbcfc40274ab40ecbe0119560abd"
+  sha256: 4069fc9dd220f0e189de4ac56d8e65cf97e4619544ce249466bd7f3abd7b41fe
```

Comment:
New conda-skeleton uses "" for the checksum but that breaks the bot!

Ping @conda-forge/bot team. We can: fix conda-skeleton to avoid the quotes, the linter to warn about the issue, or the bot to accept it.